scripts/testnet/infra/terraform.tfstate

```diff
@@ -0,0 +1,71 @@
+{
+  "version": 4,
```

Comment:
great question. Honestly, I think the best practice would be to store the state using a terraform cloud account. It's free and pretty easy to setup.And to the second question, I *think* it will error out if we try and run a command from an outdated state - not positive though. 
One thing to talk through today might be how we separate out different deployments. I'm hoping just the variable change acts as the equivalent of having a `main.tf` if an entirely different file (in that it treats them as separate terraform modules/resources) - but definitely something to double check